packages/web-components/fast-foundation/src/data-grid/data-grid-cell.ts

```diff
@@ -0,0 +1,310 @@
+import {
+    attr,
+    FASTElement,
+    observable,
+    html,
+    HTMLView,
+    ViewTemplate,
+} from "@microsoft/fast-element";
+import {
+    keyCodeEnter,
+    keyCodeEscape,
+    keyCodeFunction2,
+} from "@microsoft/fast-web-utilities";
+import { ColumnDefinition } from "./data-grid";
+
+const defaultCellContentsTemplate: ViewTemplate = html<DataGridCell>`
+    <template>
+        ${x =>
+            x.rowData === null ||
+            x.columnDefinition === null ||
+            x.columnDefinition.columnDataKey === null
+                ? null
+                : x.rowData[x.columnDefinition.columnDataKey]}
+    </template>
+`;
+
+const defaultHeaderCellContentsTemplate: ViewTemplate = html<DataGridCell>`
+    <template>
+        ${x =>
+            x.columnDefinition === null
+                ? null
+                : x.columnDefinition.title === undefined
+                ? x.columnDefinition.columnDataKey
+                : x.columnDefinition.title}
```

Comment:
@EisenbergEffect here's a thought, would we conceivably want a kind of logging system during development to determine if data structures are missing for a given component? It seems like that would be useful here.

Reply:
Yes, that would be nice. We'd need to somehow address the issue of debug builds first I think. Otherwise, we could very quickly blow out our library size targets and hurt perf in critical areas.